pyproject.toml

```diff
@@ -72,6 +72,7 @@ numpy = { version = "<1.22", python = "~3.7" } # 1.22 doesn't support python 3.7
 requests = ">=2.23.0"
 nvsmi = "^0.4.2"
 psutil = ">=5.4.8" # Google Colab comes with 5.4.8 by default. If it gets upgraded an error message is shown saying the runtime needs to be restarted.
+lazy-loader = {git = "https://github.com/layerai/lazy_loader.git", rev = "26aa52ab057afb0c3f85ffc64f220eae0e9a222c"}
```

Comment:
Had to [fork](https://github.com/layerai/lazy_loader), because [lazy_loader](https://github.com/scientific-python/lazy_loader) only installs to `python >= 3.8`, but there were no blockers to [lower the requirement](https://github.com/layerai/lazy_loader/commit/26aa52ab057afb0c3f85ffc64f220eae0e9a222c#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R8).